addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -34,12 +40,15 @@ class ScEnvironmentCard extends React.Component {
   render() {
     const env = this.environment;
     const state = env.state;
+    const indexesStore = this.indexesStore;
+    const indexes = indexesStore.getIndexes();
 
     return (
       <>
         {this.renderStatus(state)}
         {this.renderTitle(env)}
         {this.renderError(env)}
+        {this.renderWarning(env, indexes)}
```

Comment:
It looks like we're passing in `indexes` to this method, but this method only have one parameter in its method signature.

Reply:
I was planning to use it later so that an Account name can be displayed as well in the warning message. I'll remove it if I don't get around to doing this as part of this PR.

Reply:
Removed for now.